packages/test/local-server-tests/src/test/stagingMode.spec.ts

```diff
@@ -37,6 +38,8 @@ class RootDataObject extends DataObject {
 			? -1
 			: RootDataObject.instanceCount++;
 
+	private readonly containerRuntimeExp: IContainerRuntimeBaseExperimental = this.context
+		.containerRuntime satisfies IContainerRuntimeBaseExperimental;
```

Comment:
The `satisfies` may be optional, I don't get any build errors when removing it.  Here and elsewhere.